### ISSUE TITLE
Invalid parameter while pairing with VIZIO Tv

### ISSUE THREAD

**Issue by VishV-Android** [exiva/Vizio_SmartCast_API#41]
By using this URL -> https://192.168.1.10:7345/pairing/start (put request type)  , I am able to get **4 digit pin** on the TV screen.
But after then when I use this -> https://192.168.1.10:7345/pairing/pair (put request type) with 4 body parameter which are,

{"key":"DEVICE_ID","value":"57b582c6-42c4-4b96-b335-7c6de689aac2"},
{"key":"RESPONSE_VALUE","value":"1898"},
{"key":"PAIRING_REQ_TOKEN","value":"870080"},
{"key":"CHALLENGE_TYPE","value":"1"}

I am passing this parameter as body but I am getting this as response ,
{
    "STATUS": {
        "RESULT": "INVALID_PARAMETER",
        "DETAIL": "Invalid parameter"
    }
}
Note: This request made in postman but still not working. 
Is anyone getting same issue?

**Comment by enskyeing** [comment]
Did you ever figure this out?

**Comment by VishV-Android** [comment]
@enskyeing Yes, I have figured this out. This request is not working in the Postman but when I requested using OkHttp3 in my project then it was working fine.
